app/src/main/res/values/strings.xml

```diff
@@ -692,12 +692,12 @@
     <string name="cd_dns_proxy_error_text_1">Invalid IP address</string>
     <string name="cd_dns_proxy_error_text_2">Port range should be from 1024–65535</string>
     <string name="cd_dns_proxy_error_text_3">Invalid port</string>
-    <string name="cd_dns_crypt_dialog_title">Add DNSCrypt Resolver or Relay</string>
+    <string name="cd_dns_crypt_dialog_title">Add DNSCrypt resolver or relay</string>
```

Comment:
uppercase preferred.

Reply:
Not a proper noun. https://www.dnscrypt.org/ uses lowercase.
To be completely on the safe side:
```suggestion
    <string name="cd_dns_crypt_dialog_title">Add DNSCrypt resolver or -relay</string>
```
Avoids ___ resolver, or (nothing) just a relay.